R/compiled.R

```diff
@@ -62,7 +91,8 @@ clean_gcov <- function(path) {
 
 run_gcov <- function(path, quiet = TRUE,
                       gcov_path = getOption("covr.gcov", ""),
-                      gcov_args = getOption("covr.gcov_args", NULL)) {
+                      gcov_args = getOption("covr.gcov_args", NULL),
+                      gcov_br = FALSE) {
```

Comment:
so now we have `include_branches`, `br` and `gcov_br` :-)

Reply:
Now that you pointed it out, I think `parse_gcov`'s `br` option should also be called `gcov_br` to be consistent and reduce confusion.